editor/sidebar/style.scss

```diff
@@ -0,0 +1,9 @@
+.editor-sidebar {
+	position: fixed;
+	right: 0;
+	top: 32px + $header-height;
```

Comment:
We'll need some breakpoint styling for this. Not sure if we wanted to do that here.